frontend/arrangor-flate/app/routes/$orgnr.utbetaling.$id.detaljer.tsx

```diff
@@ -158,3 +181,55 @@ function getUtbetalingTypeNavn(type: UtbetalingType) {
       return "Utbetaling for investering";
   }
 }
+
+interface DeltakerModalProps {
+  utbetaling: ArrFlateUtbetaling;
+  relevanteForslag: RelevanteForslag[];
+  deltakerlisteUrl: string;
+}
+
+function DeltakerModal({ utbetaling, relevanteForslag, deltakerlisteUrl }: DeltakerModalProps) {
+  const modalRef = useRef<HTMLDialogElement>(null);
+  if (utbetaling.beregning.type === "FRI" || utbetaling.erTolvUkerEtterInnsending) {
+    return null;
+  }
+  return (
+    <HStack gap="2">
+      <Button variant="secondary" size="small" onClick={() => modalRef.current?.showModal()}>
+        Åpne deltakerliste
```

Comment:
Det kan hende det er litt uheldig å kalle dette for "deltakerliste" (mulig det kan blandes med komet sin løsning? Men vet ikke). Alternativt noe ala. "Vis deltakere" eller "Vis beregning"?